projects/packages/search/package.json

```diff
@@ -56,7 +56,9 @@
 		"preact": "10.5.15",
 		"q-flat": "1.0.7",
 		"qss": "2.0.3",
-		"react": "17.0.2",
+		"react": "16.14.0",
```

Comment:
Why are you downgrading React?

Reply:
Neither `@wordpress/block-editor` nor `@wordpress/components` accept React 17 as a valid peer dependency. I downgraded here to mitigate any risk of breaking changes at this time.

That said, we may need to revisit this change due to Jetpack components relying on React 17.